ReadMe.md

```diff
@@ -83,73 +83,109 @@ Each `ResourceManager` is passed a base directory where searches are made from.
 #### Resource Manager Example Code
 ```C++
 #include "OP2Utility.h"
+#include <memory>
+#include <string>
+#include <vector>
 
 // Creates a ResourceManager whose base directory is EnvironmentDirectory/TestDirectory
 ResourceManager resourceManager("TestDirectory");
 
 // Load a map into memory without extracting the file to disk if it is contained in an archive.
-Map map(resourceManager.GetResourceStream("MesaMissions.map"));
+std::unique_ptr<Stream::BidirectionalReader> mapStream = resourceManager.GetResourceStream("MesaMissions.map");
+Map op2Map = Map::ReadMap(*mapStream);
 
-// Same as above, but only look for an already extracted file
-map(resourceManager.GetResourceStream("MesaMissions.map", false));
+// Same as above, but prevent searching archives for given file
+op2Map = Map::ReadMap(*resourceManager.GetResourceStream("MesaMissions.map", false));
```

Comment:
I would prefer the parameter be an enum for clarity. I'm a bit on the fence whether we should support a function that does not search vol contents which is partially why I haven't touched it. Right now it affects at least OP2MapImager as it has a setting to prevent rendering maps within a volume, although that option could be removed from OP2MapImager.

Reply:
You may be on to something there. I'm not really sure why we would want to not search loaded VOL files. Perhaps we should defer to a separate issue.

For OP2MapImager,  I suppose it could just not load VOL files? Though perhaps there are other weaknesses if the `ResourceManager` API design. I've been meaning to convert that to more of a `VirtualFilesystem`, with more control of loading/mounting things into the filesystem.